src/modules/navigation/partials/NavigationTableItem.vue

```diff
@@ -20,7 +20,7 @@
 
 		<template #counter>
 			<NcCounterBubble v-if="canReadData(table)">
-				{{ n('tables', '%n row', '%n rows', table.rowsCount, {}) }}
+				{{ n('tables', ' %n ',table.rowsCount) }}
```

Comment:
```suggestion
				{{ table.rowsCount }}
```

As we don't need any translation of that anymore